aip/general/0001.md

```diff
@@ -28,6 +28,27 @@ serve as the source of truth for API-related documentation at Google and the
 means by which API teams discuss and come to consensus on API guidance. AIPs
 are maintained as Markdown files in the [AIP GitHub repository][].
 
+## Scope
+
+It doesn't make sense for *every* Google API to follow the AIPs. Some relevant
```

Comment:
It may help to start with base assumptions about what types of APIs the AIPs are for: public APIs with a wide variety of external clients (examples are in https://google.aip.dev/9#client).